cluster/examples/kubernetes/ceph/common-openshift.yaml

```diff
@@ -0,0 +1,728 @@
+###################################################################################################################
```

Comment:
Remind me why we need a common-openshift.yaml? I would not want to create a new one just for the different namespace. Some script could replace the namespace for use in downstream yamls.

Reply:
If we do decide we need it, we would also need an entry in the [examples](https://github.com/rook/rook/blob/master/Documentation/ceph-examples.md#common-resources) help topic.

Reply:
There are few things that are happening,
1. The namespace and it's labelling for monitoring
2. RBAC to create Prometheus resources
3. RBAC for prometheus instance of openshift to monitor the cluster

If we could somehow make place for it in common.yaml, it would be great.

Reply:
In common.yaml we do not add labels. That is the only major difference that I see.

Reply:
Does it make sense to add the labels in `common.yaml` then?

Reply:
`common.yaml` takes care of namespace creation. So, it shouldn't be an issue adding labels as metadata.

Now, I am thinking, maybe we should just remove the differentiation. We'll maintain a more generic configuration in `common.yaml` and if user has any specific use case s/he can follow respective documentation.